src/sdk/pynni/nni/compression/torch/compressor.py

```diff
@@ -158,7 +191,6 @@ class Pruner(Compressor):
 
     def __init__(self, model, config_list):
         super().__init__(model, config_list)
-        self.mask_dict = {}
 
     def calc_mask(self, layer, config):
```

Comment:
input arguments are not consistent with other places

Reply:
fixed